### DISCUSSION TITLE
Whether the image of toga.ImageView can be zoomed in or out?

### DISCUSSION THREAD

**Discussion by ccinsz** [beeware/briefcase#1440] [Q&A]
Whether the image of toga.ImageView can be zoomed in or out? If it can, how to do it.

**Comment by freakboy3742** [comment]
It's not clear what you mean by this. The image will render at the size determined by the layout. That might involve scaling the image. 

If you mean something like "pinch to zoom" on an image - no, it doesn't support that. ImageView is an analog of `<img>` on a web page - it puts an image on the page. If you want to zoom the image, that's a UI interaction you need to build. 

At present, we don't have any handlers for "pinch" type actions.

**Reply by ccinsz** [reply]
Got it. Thank you for your help.
